### PR TITLE
switch from chainlink-relay to chainlink-common

### PR DESCRIPTION
**Pull request by jmank88**
https://smartcontract-it.atlassian.net/browse/BCF-2784

Requires:
- https://github.com/smartcontractkit/chainlink-common/pull/243
- https://github.com/smartcontractkit/chainlink-cosmos/pull/385
- https://github.com/smartcontractkit/chainlink-solana/pull/555
- https://github.com/smartcontractkit/chainlink-starknet/pull/335

### CODE REVIEW

File: core/cmd/cosmos_node_commands_test.go [hunk 1]
```diff
@@ -9,9 +9,9 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	relaycfg "github.com/smartcontractkit/chainlink-common/pkg/config"
```
Comment (prashantkumar1982) [inline core/cmd/cosmos_node_commands_test.go:12]: nit: Do we just want to get away from the relay name here?
And use commoncfg?

File: core/services/chainlink/config_test.go [hunk 2]
```diff
@@ -16,9 +16,9 @@ import (
 
 	ocrcommontypes "github.com/smartcontractkit/libocr/commontypes"
 
+	relayassets "github.com/smartcontractkit/chainlink-common/pkg/assets"
```
Comment (prashantkumar1982) [inline core/services/chainlink/config_test.go:19]: maybe here too

File: core/services/chainlink/relayer_chain_interoperators_test.go [hunk 3]
```diff
@@ -9,9 +9,9 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	relaycfg "github.com/smartcontractkit/chainlink-common/pkg/config"
```
Comment (prashantkumar1982) [inline core/services/chainlink/relayer_chain_interoperators_test.go:12]: this too, and possibly more in other files

File: core/cmd/solana_transaction_commands_test.go [hunk 4]
```diff
@@ -29,7 +29,7 @@ func TestShell_SolanaSendSol(t *testing.T) {
 	url := solanaClient.SetupLocalSolNode(t)
 	node := solcfg.Node{
 		Name: ptr(t.Name()),
-		URL:  utils.MustParseURL(url),
+		URL:  config.MustParseURL(url),
```
Comment (reductionista) [inline core/cmd/solana_transaction_commands_test.go:32]: Huh, weird that we even have this in multiple locations. Seems like Solana should be importing this from `chainlink-commons` right?

**Review by reductionista — DISMISSED** [review]

**Review by prashantkumar1982 — APPROVED** [review]

**Pull request merged**